src/AspNetCore/Inc/aspnetcore_event.h

```diff
@@ -0,0 +1,551 @@
+
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#ifndef __ASPNETCOREEVENT_H__
+#define __ASPNETCOREEVENT_H__
+/*++
+
+    Module Name:
+
+        aspnetcore_event.h
+
+    Abstract:
+
+        Header file has been generated from mof file containing 
+        IIS trace event descriptions
+
+--*/
+
+//
+// Start of the new provider class WWWServerTraceProvider,
+// GUID: {3a2a4e84-4c21-4981-ae10-3fda0d9b0f83}
+// Description: IIS: WWW Server
+//
+
+class WWWServerTraceProvider
+{
+public:
+    static
+    LPCGUID
+    GetProviderGuid( VOID )
+    // return GUID for the current event class
+    {
+        static const GUID ProviderGuid = 
+          {0x3a2a4e84,0x4c21,0x4981,{0xae,0x10,0x3f,0xda,0x0d,0x9b,0x0f,0x83}};
+        return &ProviderGuid;
+    };
+    enum enumAreaFlags
```

Comment:
Do we plan to add more flags here than just ANCM? Seems odd to have with only one value.

Reply:
this is auto generated file with some IIS tool